airflow_valohai_plugin/operators/valohai_download_execution_outputs_operator.py

```diff
@@ -0,0 +1,54 @@
+import os
+import re
+from urllib.request import urlretrieve
+import logging
+
+from airflow.models import BaseOperator
+from airflow.configuration import AIRFLOW_HOME
+
+
+class ValohaiDownloadExecutionOutputsOperator(BaseOperator):
+    """
+    Downloads outputs locally from a ValohaiSubmitExecutionOperator
+    tasks that stored outputs details in XCOM.
+
+    Args:
+        output_task_id (str): id of the ValohaiSubmitExecutionOperator
+            task that produced the outputs.
+        output_path (str): relative path to AIRFLOW_HOME where to
+            store the outputs locally.
+        output_name_pattern (str, optional): name or regex pattern to filter
+            output names to donwload.
```

Comment:
Unless I'm mistaken, we're doing regex filtering so we should be really clear about that

Reply:
Should we have `output_name` and `output_name_pattern` to allow both options and have a clear barrier between those two!

Reply:
Commented below. I may then change that to `output_name` and maybe add an option `is_regex` that by default will be `False` to avoid confusion. Thanks.

Reply:
Thanks. at the end I went for for your suggestion and created `output_name` and `output_name_pattern`